File: Java/SETPractice/src/main/java/DesignPattern/FacadePattern/Restaurant.java
package DesignPattern.FacadePattern;

public class Restaurant implements IRestaurant{
    @Override
    public void PrepareMenu(String menu) {
        System.out.println("Preparing the Menu " + menu);
    }

    @Override
    public void ServeMenu(String menu) {
        System.out.println("Serving the Menu " + menu);
    }
}

File: DotNet/SETPractice/DesignPattern/FacadePattern/Restaurant.cs
namespace DesignPattern.FacadePattern
{
    public class Restaurant : IRestaurant
    {
        public void PrepareMenu(string menu)
        {
            Console.WriteLine($"Preparing the Menu {menu}");
        }

        public void ServeMenu(string menu)
        {
            Console.WriteLine($"Serving the Menu {menu}");
        }
    }
}
